### ISSUE TITLE
Check white noise generation

### ISSUE THREAD

**Issue by pausz** [BrainDynamicsUSYD/nftsim#89]
The current implementation of White noise assumes we're working with a square grid in order to compute `deltax`. If the grid is rectangular, the current scaling of the noise will result in an incorrect value of deltax.
 
https://github.com/BrainDynamicsUSYD/nftsim/blob/master/src/timeseries.cpp#L330

Related issues #90 and #151.

Current behaviour if the grid has 450 nodes and Longside is 30 (nodes), then 
+ deltax = 0.0166667 m (according to Wave)
+ deltax = 0.0235702 m (according to TimeSeries-White and WhiteCoherent)

```
Time: 8 Deltat: 9.765625e-04
Nodes: 64

    Connection matrix:
From:  1 
To 1:  0


Population 1: Stimulation
Length: 0.5
    Stimulus: White - Onset: 0 Mean: 0 PSD: 1e-05

Output: Node: All Start: 0
Population: 1
Dendrite:  
Propagator:
Coupling: 
```

```Matlab
x = out.data{1}; % out is the nftsim struct
[Pxx, F] = pwelch(x, [], [], 512, 1/9.765625e-04, 'onesided');
```
![screenshot from 2018-02-15 15-51-17](https://user-images.githubusercontent.com/1563810/36241103-fb3f835e-125f-11e8-9e1a-95c97918f570.png)

Changing the time step to 4.8828e-04 and repeating the analysis we obtain:
![screenshot from 2018-02-15 15-58-20](https://user-images.githubusercontent.com/1563810/36241245-beed9e3a-1260-11e8-8ffe-b4fd8a94d390.png)

Bottom line: the scaling of the variance of the underlying normal distribution used to generate a sequence of random numbers, really depends on what property of the resulting signal one needs to keep constant (eg, PSD or average power). This scaling might be at odds with *interpretations* of what that sequence of random numbers  is (eg, increments of a Wiener process, just and input signal with a really broad band), etc.
